ubiquibot-config-default.ts

```diff
@@ -90,5 +91,8 @@
   "enable-access-control": {
     "label": false,
     "organization": true
-  }
+  },
+  "stale-bounty-time":""
```

Comment:
This doesn't look right. Why did you remove "0d"

Reply:
0d wasn't [there](https://github.com/ubiquity/ubiquibot/blob/development/ubiquibot-config-default.json#L92) originally, thanks for mentioning. I will update it.